src/components/filters/Filters.jsx

```diff
@@ -0,0 +1,194 @@
+import { useState,useEffect } from "react";
+import "./Filters.css";
+
+export const Filters = () => {
+  /** state used to toggle between responsive filter and normal filter **/
+  const [filtersClassName, setFiltersClassName] = useState(
+    "filters-wrapper flex-column"
+  );
+  const checkDeviceWidth = () => {
+    const deviceWidth = window.innerWidth;
+    if (deviceWidth <= 897) {
+      setFiltersClassName("hide-filters");
+    } else {
+      setFiltersClassName("filters-wrapper flex-column");
+    }
+  };
+  useEffect(() => window.addEventListener("resize", checkDeviceWidth), []);
+  const showFilters = () => {
+    setFiltersClassName(
+      filtersClassName ===
+        "filters-wrapper filters-wrapper-responsive flex-column"
+        ? "hide-filters"
+        : "filters-wrapper filters-wrapper-responsive flex-column"
+    );
+  };
```

Comment:
Liked that you made a function to handle the responsive view of the filter component and use it accordingly